source/agora/test/PeriodicCatchup.d

```diff
@@ -94,7 +94,7 @@ private class NodeManager (): TestAPIManager
         if (this.nodes.length < 1)  // Use first node as test node
         {
             assert(conf.validator.enabled);
-            log.trace("Create node {} as cathup test node", this.nodes.length);
+            log.trace("Create node #{} ({}) as cathup test node", this.nodes.length, conf.validator.key_pair.address);
```

Comment:
Own commit

Reply:
This was extracted into separate commit and merged in PR https://github.com/bosagora/agora/pull/2288